File: portafolio-backend/src/main/java/com/portafolio/portafoliobackend/dtos/TblPortafolioDTO.java
package com.portafolio.portafoliobackend.dtos;

import java.util.Date;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class TblPortafolioDTO {
    
    // CAMPO IDENTIFICADOR DEL PORTAFOLIO
    private Long idPortafolio;

    // CAMPO NOMBRE DE PORTAFOLIO
    private String noPortafolio;

    // CAMPO DESCRIPCIÓN DEL PORTAFOLIO
    private String dePortafolio;
    
    // CAMPO IMAGEN DEL PORTAFOLIO
    private String imgPortafolio;

    // CAMPO ESTADO DEL REGISTRO. LOS POSIBLES VALORES SON: "1" = ACTIVO Y "0" = INACTIVO
    private String esRegistro;

    // CAMPO USUARIO CREADOR
    private String usCreacion;

    // CAMPO TERMINAL DE CREACIÓN
    private String ipCreacion;

    // CAMPO FECHA Y HORA DE CREACIÓN
    private Date feCreacion;

    // CAMPO USUARIO MODIFICADOR
    private String usActualizacion;

    // CAMPO TERMINAL DE MODIFICACIÓN
    private String ipActualizacion;

    // CAMPO FECHA Y HORA DE MODIFICACIÓN
    private Date feActualizacion;
}

File: portafolio-frontend/src/app/models/TblPortafolioDTO.ts
export class TblPortafolioDTO {

  // CAMPO IDENTIFICADOR DEL PORTAFOLIO
  idPortafolio: number;

  // CAMPO NOMBRE DE PORTAFOLIO
  noPortafolio: string;

  // CAMPO DESCRIPCIÓN DEL PORTAFOLIO
  dePortafolio: string;

  // CAMPO IMAGEN DEL PORTAFOLIO
  imgPortafolio: string;

  // CAMPO ESTADO DEL REGISTRO. LOS POSIBLES VALORES SON: "1" = ACTIVO Y "0" = INACTIVO
  esRegistro: string;

  // CAMPO USUARIO CREADOR
  usCreacion: string;

  // CAMPO TERMINAL DE CREACIÓN
  ipCreacion: string;

  // CAMPO FECHA Y HORA DE CREACIÓN
  feCreacion: Date;

  // CAMPO USUARIO MODIFICADOR
  usActualizacion: string;

  // CAMPO TERMINAL DE MODIFICACIÓN
  ipActualizacion: string;

  // CAMPO FECHA Y HORA DE MODIFICACIÓN
  feActualizacion: Date;
}
